src/App.js

```diff
@@ -103,8 +103,8 @@ const footerProps = data;
         <Navigator mainMenu={mainMenu} topMenuId={topMenuId.toString()} />
         <Switch>
             <Route exact path="/" render={() => ( <HomePage homePageProps={homeProps} classname="main" /> )}/>
-            <Route path="/about-page" render={() =>  <About aboutProps={aboutProps} sideMenu={subMenu} styleName="main" /> }/>
-            <Route path="/how-to" render={() =>  <HowPage styleName="main"/> }/>
+            <Route path="/about-open-referral" render={() =>  <About aboutProps={aboutProps} sideMenu={subMenu} styleName="main" /> }/>
+            <Route path="/how-it-works" render={() =>  <HowPage styleName="main"/> }/>
```

Comment:
These 2 links currently don't take you to the requested page when clicked

Reply:
I thought they are, I am getting:

![image](https://user-images.githubusercontent.com/54143283/115233523-edbb7980-a10f-11eb-90e0-ce4b993deae2.png)

and: 

![image](https://user-images.githubusercontent.com/54143283/115233568-fb70ff00-a10f-11eb-870c-431259368117.png)